lib/config/presets/internal/monorepo.ts

```diff
@@ -3,20 +3,20 @@ import type { Preset } from '../types';
 
 const repoGroups = {
   'ag-grid': 'https://github.com/ag-grid/ag-grid',
```

Comment:
@HonkingGoose  can we sort them alphanumeric?

Reply:
Feel free to put them in a alphanumeric order! 😄